src/Panel/RoutesPanel.php

```diff
@@ -30,6 +31,11 @@ class RoutesPanel extends DebugPanel
      */
     public function summary()
     {
+        $appClass = Configure::read('App.namespace') . '\Application';
+        if (class_exists($appClass, false) && !Router::$initialized) {
```

Comment:
https://github.com/cakephp/cakephp/blob/80e225590d06e7e40ea6aab21571269c7873c3a5/src/Routing/Router.php#L41-L42

Reply:
@saeideng We'll have $initialized around until 4.0.0, as we can't remove the autoloading of routes until the.